addons/better-quoter/module.js

```diff
@@ -0,0 +1,203 @@
+let forumIdAddon = null;
+let betterQuoterAddon = null;
+let isSetup = false;
```

Comment:
Is `isSetup` ever set to true?

Reply:
Oops, looks like it isn't.
TBH it probably wouldn't cause any problems if it ran `setup()` twice, but I'll go fix that real quick anyway.